README.md

```diff
@@ -13,10 +13,10 @@ I have done the first one for you, as an example.
 
 | $f(n)=\ldots$              | compared to | $g(n)=\ldots$          |
 |----------------------------|:-----------:|------------------------|
-| $f(n)=n$                   | 1           | $g(n)=2n^2 + n$        |
-| $f(n)= 10n + 3\log_{15} n$ | 2           | $g(n)= 4n - 2\log_2 n$ |
-| $f(n) = 2n^5$              | 2            | $g(n) = 5n^2$          |
-| $f(n)=\log_{10} \left(n^{10}\right)$ | 4  | $g(n)=n$ |
-| $f(n)= 4n^5 $ | 2  | $g(n)= 5n^4$ |
-| $f(n) = 10^{256}$ | 1  | $g(n) = \log n$ |
-| $f(n)= n^2 $ | 1  | $g(n)= 2^n$ |
+| $f(n)=n$                   | 1           | $g(n)=2n^2 + n$        |      // f(n) gorws slower than g(n) and will never overtake it.
+| $f(n)= 10n + 3\log_{15} n$ | 2         | $g(n)= 4n - 2\log_2 n$ |      // f(n) grows faster than g(n) and will not be bounded above by g(n) for large values of n, fitting the description of being in Big Omega(g(n)) but not in Big O(g(n))
```

Comment:
Hmmm, are you sure about this? Could we not multiply $g(n)$ by a constant to make it larger than $f(n)$?